exp/constraints.lisp

```diff
@@ -366,18 +369,18 @@
 (defconstraint preprocessing-4-modexp-log (:perspective macro :guard IS_MODEXP_LOG)
   (callToISZERO 4 0 (raw_lead_hi)))
 
-(defun (raw_hi_part_is_zero)
+(defun (raw_lead_hi_is_zero)
   (shift preprocessing/WCP_RES 4))
 
 ;; 5
 (defconstraint preprocessing-5-modexp-log (:perspective macro :guard IS_MODEXP_LOG)
-  (callToISZERO 5 0 (padded_base_2_log)))
+  (callToISZERO 5 0 (trim_acc)))
 
-(defun (padded_base_2_log)
-  (+ (* 8 (nbytes_excluding_leading_byte)) (nbits_of_leading_byte_excluding_leading_bit)))
+(defun (trim_acc_is_zero)
+  (shift preprocessing/WCP_RES 5))
 
 (defun (trivial_trim)
-  (shift preprocessing/WCP_RES 5))
+  (eq! 1 1))
```

Comment:
What's going on here ? We don't need `trivial_trim` anymore, do we ?

Reply:
ops, it was a placeholder to compile during WIP. I'll remove it!